pages/authzed/api/_meta.json

```diff
@@ -5,9 +5,7 @@
     "newWindow": true
   },
   "http-api": {
-    "title": "HTTP API Reference",
-    "href": "https://www.postman.com/authzed/workspace/spicedb/overview",
-    "newWindow": true
+    "title": "HTTP API Reference"
   },
   "cloud-api": {
```

Comment:
not a blocker / for a follow up: now that we embed the SpiceDB HTTP API using Swagger, we should probably do the same with the Authzed Cloud API